src/views/registration/attendees/fields/checkbox.php

```diff
@@ -39,7 +39,7 @@ class="ticket-meta"
 					name="tribe-tickets-meta[<?php echo esc_attr( $attendee_id ) ?>][<?php echo esc_attr( $slug ); ?>]"
 					value="<?php echo esc_attr( $option ); ?>"
 					<?php echo $required ? 'required' : ''; ?>
-					<?php checked( true, in_array( $slug, $value ) ); ?>
+					<?php checked( true, in_array( $slug, $value ) ); ?>>
```

Comment:
Looks good however I would use explicit self closing to make a bit more clear and place it on the next line: 

```php
/>
```